### PR TITLE
Mark release 2.7.2

### PR DESCRIPTION
**Pull request by wyrzyk**

### CODE REVIEW

**Review by afaruga-atlassian — APPROVED** [review]

**Comment by afaruga-atlassian** [comment]
```
TestDualConnection > shouldUseMainAfterSettingTimeoutUpperCaseInCompatibilityMode() FAILED
    java.lang.AssertionError at TestDualConnection.java:1696
```
😞

File: src/test/java/com/atlassian/db/replica/api/TestDualConnection.java [hunk 1]
```diff
@@ -1677,25 +1677,6 @@ public void shouldKeepUsingReplicaAfterSettingTimeoutUpperCase() throws SQLExcep
...
-        ).compatibleWithPreviousVersion()
-            .build();
-
-        try (Statement statement = connection.createStatement()) {
-            statement.execute("SET statement_timeout to 30000");
-        }
-
-        connection.prepareStatement(SIMPLE_QUERY).executeQuery();
-
-        assertThat(connectionProvider.getProvidedConnectionTypes())
```
Comment (afaruga-atlassian) [inline src/test/java/com/atlassian/db/replica/api/TestDualConnection.java:1695]: this isn't a true anymore?

**Review by afaruga-atlassian — APPROVED** [review]

File: src/test/java/com/atlassian/db/replica/api/TestDualConnection.java (hunk 1 above)
Comment (wyrzyk) [reply src/test/java/com/atlassian/db/replica/api/TestDualConnection.java:1695]: Yes. Cleaning the compatibility flag usages is a part of releasing process -> https://github.com/atlassian-labs/db-replica/blob/master/docs/release/releasing.md

Comment (afaruga-atlassian) [reply src/test/java/com/atlassian/db/replica/api/TestDualConnection.java:1695]: I know this.
That's why I'll remove `compatibleWithPreviousVersion()` from `given` section, but left the assertions

Comment (wyrzyk) [reply src/test/java/com/atlassian/db/replica/api/TestDualConnection.java:1695]: The test for no compatibility mode is already there. We don't need a redundant one.

**Pull request merged**